tests/blazorwasm2/Server/Startup.cs

```diff
@@ -24,7 +24,7 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             // Adds Microsoft Identity platform (AAD v2.0) support to protect this Api
-            services.AddMicrosoftWebApiAuthentication(Configuration, "AzureAd");
+            services.AddMicrosoftIdentityPlatformWebApiAuthentication(Configuration, "AzureAd");
```

Comment:
Could we use this for consistency?
```suggestion
            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
                    .AddMicrosoftIdentityPlatformWebApi(Configuration, "AzureAd")
```

Reply:
That's our test app. We can use our shortcuts (we have to test them)
For the templates it's another story